TEMPLATE_LIST.md

```diff
@@ -2,6 +2,26 @@
 
 Please unfold any entry to see a screenshot.
 
+New in 2.0.3:
+  * Coriolis System "yzecoriolis"
+  * New versions of Fallout 2d20 "fallout". See the Fallout section for details.
+  * Alien RPG has been updated to include Building Better Worlds Careers.
+  * Added the special statuses from the Starfinder "sfrpg" system.
+  * Note from Michael Card (Emo Cthulhu): I have added versions without Character Names to most of my templates, not just Dragonbane. Why?
+    - Since hovering over the actor portrait shows the name, it's nice to have a smaller, more compact table.
+    -  As a GM I know my players and what character they are portraying, so it's not needed. I thought other GMs might feel the same way. I see the portrait and that's enough. On the rare occasion that I'm running a game for unkown people, the name can be seen via hovering over the portrait.
+    -  I have not touched templates submitted by others or that were done by request (so far, SWADE, and Fallout 2d20), this includes the 5e templates.
+    -  I did not change the Starfinder templates, as there's a lot of other info in that column.
+    -  Have a look at the example images and pick the one you want to put in your `totm` folder.
+
+# Coming Soon:
+  - Star Trek Adventures "sta"
+  - Cyberpunk RED "cyberpunk-red-core"
+  - Dragon AGE "dragon-age"
+  - PF1E "pf1"
+  - PF2E "pf2e"
+  - Cities Without Number "wwn"
+
```

Comment:
Rewrite/remove this section. I want to keep the template_list and all other documents agnostic and informative for the most part.  No personal asides, no coming soon, and definitely no 'new in Version X (that's what the changelog is for)'

Reply:
Gotta say some of it needs to be said somwhere. Hell, all of it. And the changelog doesn't show shit, only shows 2.0.1 for me. And doesn't show anything I did or added, just your issues.

Reply:
I suggest you read up on Conventional Commits, as that's how the CHANGELOG is generated.
 https://www.conventionalcommits.org/en/v1.0.0/#summary

> only shows 2.0.1 for me

There was an issue with the build system and we're a number off. 2.0.1 is the last release but called 2.0.2 because I fucked something up.

The changelog updates when a new release is made. As it stands now, when I squash these changes in from a PR that isn't mine, I have to manually insert the conventional commit lines or manually edit the changelog to reflect things.

Also if it's more comfortable, we can just remove the template data from the repository, and you can make a repo for your templates that can be linked to from the README or TEMPLATE_LIST. Then you can have separate changelogs on your side.  Or we can have required template submission data, like 'system version #', 'expansions required' etc., that people have to fill out to add a template to the repo.  I just don't want the TEMPLATE_LIST to become cluttered with irrelevant data and used as a changelog file for 30 templates.